assets/js/translator.js

```diff
@@ -447,12 +447,33 @@ function getPairs() /*: JQueryPromise<any> */ {
             }
         });
 
-        for(var srcLang in pairs) {
-            // Default for new users is first available pair
-            curSrcLang = srcLang;
-            curDstLang = pairs[srcLang][0];
-            break;
+        // Default for new users is first available Browser Preference Language pair
+        var brwsLangs = navigator.languages; // Chrome, Mozilla and Safari
+        var prefLang;
+        $.each(brwsLangs, function(index) {
+            if(languages[brwsLangs[index].substr(0, 2)] !== undefined) {
```

Comment:
Why are you always taking the substring? There are some languages that don't have 2-letter codes.

Reply:
https://www.metamodpro.com/browser-language-codes in this list i didnt find any languages with 3 letter codes. I used Because when en-US is preferred language, there wont be a language.so i trimmed en

Reply:
When en-US is browser prefered language. it doesnt recognize it and goes to the next lang thats why

Reply:
I don't consider that list particularly canonical... Can you do something less brittle like splitting on `-`? I'd save that into a variable so you don't have to split twice.

Reply:
Okay. :) thats fine. I will do it. :)

Reply:
Done. :)